drivers/CoreRFIDReader/include/CoreRFIDReader.h

```diff
@@ -0,0 +1,223 @@
+		inline constexpr size_t tag_detection			   = 3;
+		inline constexpr size_t set_baudrate			   = 1;
+		inline constexpr size_t set_communication_protocol = 2;
+
+	}	// namespace expected_answer_size
+
+	namespace protocol {
+
+		inline constexpr RFIDProtocol iso15693 = {
+			.id = 0x01, .gain = std::byte {0x00}, .modulation = std::byte {0xD0}};	 // gain = 34 dB, modulation = 95%
+		inline constexpr RFIDProtocol iso14443A = {
+			.id = 0x02, .gain = std::byte {0x00}, .modulation = std::byte {0xD0}};	 // gain = 34 dB, modulation = 95%
+		inline constexpr RFIDProtocol iso14443B = {
+			.id = 0x03, .gain = std::byte {0x00}, .modulation = std::byte {0x20}};	 // gain = 34 dB, modulation = 17%
+		inline constexpr RFIDProtocol iso18092 = {
+			.id = 0x04, .gain = std::byte {0x00}, .modulation = std::byte {0x20}};	 // gain = 34 dB, modulation = 17%
+
+	};	 // namespace protocol
+
+	namespace status {
+
+		inline constexpr uint8_t communication_succeed		  = 0x80;
+		inline constexpr uint8_t error_time_out				  = 0x87;
+		inline constexpr std::array<uint8_t, 2> idn_success	  = {0x00, 0x0F};
+		inline constexpr std::array<uint8_t, 2> setup_success = {0x00, 0x00};
+		inline constexpr std::array<uint8_t, 2> tag_detection_callback {0x01, 0x02};
+
+	}	// namespace status
+
+	namespace command {
+
+		inline constexpr uint8_t send_receive = 0x04;
+
+		namespace idn {
+			inline constexpr uint8_t id		= 0x01;
+			inline constexpr uint8_t length = 0x00;
+		}	// namespace idn
+
+		namespace set_baudrate {
+			inline constexpr uint8_t id		= 0x0A;
+			inline constexpr uint8_t length = 0x01;
+		}	// namespace set_baudrate
+
+		namespace set_protocol {
+			inline constexpr uint8_t id		= 0x02;
+			inline constexpr uint8_t length = 0x02;
+		}	// namespace set_protocol
+
+		namespace set_gain_and_modulation {
+			inline constexpr uint8_t id		= 0x09;
+			inline constexpr uint8_t length = 0x04;
+		}	// namespace set_gain_and_modulation
+
+		namespace frame {
+			inline constexpr std::array<uint8_t, 16> set_mode_tag_detection {
+				rfid::settings::idle_tag_detection::tag_detection_command,
+				rfid::settings::idle_tag_detection::length,
+				rfid::settings::idle_tag_detection::wu_source,
+				rfid::settings::idle_tag_detection::enter_control[0],
+				rfid::settings::idle_tag_detection::enter_control[1],
+				rfid::settings::idle_tag_detection::wu_control[0],
+				rfid::settings::idle_tag_detection::wu_control[1],
+				rfid::settings::idle_tag_detection::leave_control[0],
+				rfid::settings::idle_tag_detection::leave_control[1],
+				rfid::settings::idle_tag_detection::wu_periode,
+				rfid::settings::idle_tag_detection::oscillator_start,
+				rfid::settings::idle_tag_detection::digital_to_analog_start,
+				rfid::settings::idle_tag_detection::digital_to_analog_data[0],
+				rfid::settings::idle_tag_detection::digital_to_analog_data[1],
+				rfid::settings::idle_tag_detection::swing_count,
+				rfid::settings::idle_tag_detection::max_sleep};
```

Comment:
rajoute une `,` après `sleep` pour que le `};` passe à la ligne et que ce soit plus lisible

```suggestion
				rfid::settings::idle_tag_detection::max_sleep,};
```